From e7dd8df8eb562d0690add0ad587e8bba72eedb9e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnaro=CC=88k?= <steven@nuclearsandwich.com>
Date: Mon, 23 Nov 2020 15:58:55 -0800
Subject: [PATCH 2/2] Directly add debian packages for added dependencies.

As noted in https://github.com/ros-infrastructure/bloom/issues/609
per-platform package.xml files are not currently read so these
dependencies need to be added to the control file as well as to the
package.xml.
---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 6d7b65c..a1d3b9d 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -2,7 +2,7 @@ Source: @(Package)
 Section: misc
 Priority: optional
 Maintainer: @(Maintainer)
-Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(BuildDepends))
+Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(['default-jdk', 'maven'] + BuildDepends))
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
-- 
2.29.2

